src/env.h

```diff
@@ -564,6 +571,7 @@ class Environment {
   debugger::Agent debugger_agent_;
 #if HAVE_INSPECTOR
   inspector::Agent inspector_agent_;
+  std::vector<v8_inspector::V8ContextInfo> contexts_;
```

Comment:
I don't think V8ContextInfo is safe to use here because it doesn't look like it's supposed to outlive a HandleScope.  The context is stored as a `v8::Local<v8::Context>` and Locals are clobbered when the HandleScope is destroyed.